types/protractor-http-mock/index.d.ts

```diff
@@ -1,6 +1,7 @@
-// Type definitions for protractor-http-mock
+// Type definitions for protractor-http-mock 0.10
```

Comment:
This is the current version of this library:
https://github.com/atecarlos/protractor-http-mock/blob/77df9c34a6cb837856fdb23e9cfb580a51ea4aec/package.json